working/1426-extension-types/feature-specification-views.md

```diff
@@ -286,6 +455,33 @@ The token `view` is made a built-in identifier.
 be the name of a type but can be the basename of a member, e.g., the
 built-in identifiers.*
 
+A few errors can be detected immediately from the syntax:
+
+If a view declaration named `View` includes a
+`<viewPrimaryConstructor>` then it is a compile-time error if the
+declaration includes a constructor declaration named `View`. (*But it
+can still contain other constructors.*)
+
+A compile-time error occurs if a view declaration declares a
+constructor which is redirecting, or a constructor which is
+generative. *So all view constructors are non-redirecting factories.*
+
+If a view declaration named `View` does not include a
+`<viewPrimaryConstructor>` then an error occurs unless it declares a
+factory constructor named `View`, that declares one required, positional
+formal parameter, and does not declare any other parameters.
+
+The _name of the representation_ in a view declaration that includes a
+`<viewPrimaryConstructor>` is the identifier specified in there. In a
+view declaration named `View` that does not include a
+`<viewPrimaryConstructor>`, the name of the representation is the name
+of the unique formal parameter of the factory constructor named `View`.
```

Comment:
This seems overly limiting to me.  It seems specifically useful to be able to construct the representation in the constructor.  Maybe use the unique field?  Or are you proposing a sort of hybrid model where the `View` constructor also declares the field (even when not a primary constructor)?  This seems unlikely to be consistent with what we do for regular classes.